rotkehlchen/icons.py

```diff
@@ -43,11 +44,12 @@ def __init__(self, data_dir: Path, coingecko: Coingecko) -> None:
         self.failed_asset_ids: Set[str] = set()
 
     def iconfile_path(self, asset: Asset) -> Path:
-        return self.icons_dir / f'{asset.identifier}_small.png'
+        return self.icons_dir / f'{urllib.parse.quote_plus(asset.identifier)}_small.png'
```

Comment:
Explained in the other comment also. The identifier can be in the URL and for the CAIPs indentifier to be there needs to be url encoded since it has `/` and `:`